src/utils/interaction-utils.js

```diff
@@ -156,11 +156,11 @@ export function getTooltipDisplayDeltaValue({
       // safely cast string
       displayDeltaValue = defaultFormatter(displayDeltaValue);
       const deltaFirstChar = displayDeltaValue.charAt(0);
-      if (deltaFirstChar !== '+' && deltaFirstChar !== '-') {
+      if (deltaFirstChar !== '+' && deltaFirstChar !== '−') {
```

Comment:
This one took some time to figure out, but tests were failing due to a change in `d3-format` which now uses a minus sign (`−`) instead of a hyphen (`-`), which according to [this issue](https://github.com/d3/d3-format/issues/62) is the correct way to provide better text alignment across different fonts.

I've made the updates to match this change and use the minus sign instead, if this is not what we want to do, according to the docs it should be possible to override the minus sign for a specific locale (relevant PR [here](https://github.com/d3/d3-format/pull/76))

Reply:
Characters that look the same in code are an incredible timesuck. At a minimum, can we use a Unicode escape sequence here (`\u2212`) instead of the bar character, plus a comment, to save some time for the next developer to deal with this code?

Reply:
Wanted to get a take on which sign we actually want to use here - revisited the comparison as there were no objections to switching to the minus sign